portfolio/render.go

```diff
@@ -226,20 +232,20 @@ func PrintRenderTable(title string, tableModel *RenderTable, writer io.Writer) {
 	for _, err := range tableModel.Errors {
 		fmt.Fprintf(writer, "[!] %v. Printing parsed information state:\n", err)
 	}
-	fmt.Fprintf(writer, "%s\n", title)
+	if title != "" {
+		fmt.Fprintf(writer, "%s\n", title)
+	}
 
-	table := tw.NewWriter(writer)
-	table.SetHeader(tableModel.Header)
-	table.SetBorder(false)
-	table.SetRowLine(true)
+	o := csv.NewWriter(writer)
+	o.Write(tableModel.Header)
```

Comment:
I think this broke the normal table render... I ran this on my machine, and it dumped CSV when I ran the command without any args.

Reply:
No longer broken. Now using the new ACBWriter interface.